### DISCUSSION TITLE
第2节课笔记(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#71] [笔记]
请在下方提交笔记链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/InternLM_Notes/blob/main/note2.md

**Comment by Alias-z** [comment]
388+http://t.csdnimg.cn/0XBCM

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135432260

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E4%BA%8C%E8%8A%82%E7%AC%94%E8%AE%B0.md

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135423752

**Comment by Vincent-luo** [comment]
566+https://www.yuque.com/vincentluo-wbw4o/xpmg2z/ugx4rx0g1yd4wx51?singleDoc# 《笔记2》

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson2/note.md

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/blob/main/Two/Note.md

**Comment by Zeno673** [comment]
567+https://github.com/Zeno673/InternLMLearning/blob/main/notes/2.%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%B6%A3%E5%91%B3Demo.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/handWrite02.md

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by jzh999** [comment]
361+https://www.yuque.com/bright-p9un8/yum3v8/rs8ruceof0454l8a?singleDoc# 《第二节、轻松玩转书生·浦语大模型趣味 Demo》

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course2.md

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/YHhmdJrBOoLKmYxu0MEcvU2onWc?from=from_copylink

**Comment by Fautines** [comment]
579+https://www.yuque.com/shijiuchennianhua-avnma/pn9l3t/qtmeapbvvr7oh6l8?singleDoc# 《day2笔记》

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/Day02.md

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9165452

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework1_note.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/f2dc8b07-0228-452a-8962-44b0d4e9d050

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/2note.md
